src/tsm/libtsm.h

```diff
@@ -320,6 +320,23 @@ enum tsm_vte_color {
 	TSM_COLOR_NUM
 };
 
+/* mouse tracking */
+#define TSM_VTE_MOUSE_MODE_X10      9
+#define TSM_VTE_MOUSE_EVENT_BTN  1002
+#define TSM_VTE_MOUSE_EVENT_ANY  1003
+#define TSM_VTE_MOUSE_MODE_SGR   1006
+#define TSM_VTE_MOUSE_MODE_PIXEL 1016
```

Comment:
Please add some comments about their usage later in state machine handling. It was a bit unclear to me at first why these particular values were chosen until I realized they are part of the control sequence.